til-domain/src/main/java/com/til/application/interview/InterviewService.java

```diff
@@ -172,6 +181,16 @@ private void createInterviewProblem(List<Long> categoryIdList, Long interviewId)
         interviewProblemRepository.saveAll(interviewProblemList);
     }
 
+    private void createSpeechInterviewProblem(List<String> questionList, Long interviewId) {
+        List<SpeechInterviewProblem> problemList = new ArrayList<>();
+        for (int i = 0; i < questionList.size(); i++) {
+            problemList.add(
+                SpeechInterviewProblem.createUnsolvedSpeechInterviewProblem(i + 1, questionList.get(i), interviewId)
+            );
+        }
+        speechInterviewProblemRepository.saveAll(problemList);
+    }
```

Comment:
P4: 이 부분도 stream을 활용해서 다음과 같이 변경해볼 수 있을 것 같습니다!
```suggestion
    private void createSpeechInterviewProblem(List<String> questionList, Long interviewId) {
        List<SpeechInterviewProblem> problemList = IntStream.range(0, questionList.size())
            .mapToObj(i -> SpeechInterviewProblem.createUnsolvedSpeechInterviewProblem(i + 1, questionList.get(i), interviewId))
            .collect(Collectors.toList());
    
        speechInterviewProblemRepository.saveAll(problemList);
    }
```

Reply:
Stream으로 변경해서 반영하겠습니다